pkg/kubectl/cmd/set/set_image.go

```diff
@@ -137,10 +138,26 @@ func (o *ImageOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []st
 		return err
 	}
 
+	var includeUninitialized bool
```

Comment:
This is a lot of duplicated code that is effectively the same.  It needs to be unified - copying and pasting this code into many places (with some differences) is a maintenance problem.